_hyperscript.tmLanguage.yaml

```diff
@@ -38,14 +38,14 @@ repository:
           \b(
             catch|if|else|otherwise
             repeat\s+forever|repeat\s+for|repeat\s+in|repeat\s+while|
-            repeat\s+until\s+event|repeat\s+until|repeat|times|
+            repeat\s+until\s+event|repeat\s+until|repeat|times|index|
             then|end|while|until|unless|for
           )\b
       - name: keyword.other.command._hyperscript
         match: &commandStarts >-
           (?x)
           \b(
-            add|remove|put|async|call|get|hide|measure|add|async|call|get|hide|measure|
+            add|remove|put|async|call|called|get|hide|measure|add|async|call|get|hide|measure|
```

Comment:
`called` belongs more in `keyword.operator.expression._hyperscript` imo. Sorry I haven't documented the scopes better